Vulmap-Linux/vulmap-linux.py

```diff
@@ -139,7 +139,7 @@ def linuxSystemInfo():
 
 def getProductList():
 	global productList
-	dpkg = "dpkg-query -W -f='${Package} ${Version}\n'"
+	dpkg = "rpm -qa --qf '%{n} %{version}\n'"
```

Comment:
This will only work for RPM-based systems but no longer for deb-based ones. 

From my point of view should the command to execute be based on the available binary on the system. `shutil.which` could help as Python 2 will be obsolete soon.